03week/airplane/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>UX Atlantic</title>
+  <link rel="stylesheet" href="./css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+
+<body>
+
+<header class="top">
+  <img src="./img/logo.png">
+  <nav>
+    <img src="./img/home-icon.png">
+    <span>Home</span>
+    <img src="./img/deal-icon.png">
+    <span>Travel Deals</span>
+    <img src="./img/schedule-icon.png">
+    <span>Flight Schedule</span>
+    <img src="./img/login-icon.png">
+    <span>Register/Login</span>
+  </nav>
+</header>
+
+<main>
+  <div id="search-box">
+    <header class="icons">
+      <img id="flights" src="./img/plane-icon.png">
+      <img src="./img/hotel-icon.png">
+      <img src="./img/car-icon.png">
+    </header>
+    <section class="inputs">
+      <div class="boxes">
+        <span class="line-one">Depart</span>
+        <span class="line-one">Arrive</span>
+        <span class="line-two">mm/dd/yyyy</span>
+        <span class="line-two">mm/dd/yyyy</span>
+      </div>
+      <div class="line-three-part-one">
+        <span>Passengers</span>
+      </div>
+      <div class="line-three-part-two">
+        <span>1</span>
+        <span id="triangle">&#x25BD;</span>
+      </div>
+      <div class="line-three-part-three">
```

Comment:
we talked about this in class already, but just to remind you, these should be different input elements (radio buttons, text inputs, and select elements)